app/pages/profile/page.jsx

```diff
@@ -6,6 +6,7 @@ import Friends from '@/components/Friends';
 import FriendsSuggestion from '@/components/FriendsSuggestion';
 import StatisticalCard from '@/components/StatisticalCard';
 import {auth,firestore,getDocs,getDoc,collection,where,query,doc} from '@/utils/firebase'
+import getAchievements from '../api/getAchievements';
```

Comment:
in the page it's self you do all the necessery callings for data fetching either on server side or client side (useEffect) and assign the data to a variable that you give to the desired component